src/Bicep.LangServer/Handlers/BicepDeploymentStartCommandHandler.cs

```diff
@@ -87,56 +87,62 @@ public override async Task<BicepDeploymentStartResponse> Handle(BicepDeploymentS
             var credential = new CredentialFromTokenAndTimeStamp(request.token, request.expiresOnTimestamp);
             var armClient = armClientProvider.createArmClient(credential, default, options);
 
-            string? parametersFileJson = null;
-
-            if (PathHelper.HasBicepparamsExension(DocumentUri.FromFileSystemPath(request.parametersFilePath).ToUri()))
+            string parametersFileJson = "{}";
```

Comment:
Any thoughts on creating an empty valid JSON string like for deployments with no parameter values?

Reply:
seems fine to me

Reply:
added a comment to clarify